packages/babel-helper-fixtures/src/index.ts

```diff
@@ -258,6 +264,7 @@ function pushTask(
   delete taskOpts.BABEL_8_BREAKING;
   delete taskOpts.DO_NOT_SET_SOURCE_TYPE;
   delete taskOpts.SKIP_ON_PUBLISH;
+  delete taskOpts.SKIP_babel7plugins_babel8core;
```

Comment:
Can you add this option to `./packages/babel-helper-fixtures/data/schema.json` ?

Reply:
Oh I had already done it :) https://github.com/babel/babel/pull/16399/files#diff-37f573cd0c03959dfce90d8debe2f678c0b30bec88b6ddf35fe383e8ccd84b30